app/assets/templates/views/wizardModal.html

```diff
@@ -57,6 +57,22 @@ <h2>Solr or Elasticsearch?</h2>
             <ul ng-if="pendingWizardSettings.searchEngine === 'solr'">
               <li>Do you see any errors when you visit your <a target="_blank" href="{{pendingWizardSettings.searchUrl}}">Solr URL directly?</a></li>
               <li>Will your request handler accept the JSONP writer type? Or does it only return XML or JSON?</li>
+              <li>
+                If you are using Solr 8.2 or later, you need to tweak the content-type being set by <code>wt=json</code>.
+                You can do this via the below curl command, replacing <i>gettingstarted</i> with your collection name:
+<pre>
+curl -X POST -H 'Content-type:application/json' -d '{
+  "update-queryresponsewriter": {
+    "name": "json",
+    "class": "solr.JSONResponseWriter",
+    "content-type": "application/json; charset=UTF-8"
```

Comment:
Should this be "text/javascript; charset=UTF-8"?

Technically application/json is correct but it won't work for JSONP when nosniff is enabled.

Reply:
Argh.  Yes.  i was testing that the emssage came up, but never tested running the code.

Reply:
@worleydl can you check again now?   Also, pushing up updated Solr 8 demo that you cant est this against as well..